lib/spoom/cli/commands/coverage.rb

```diff
@@ -90,6 +102,18 @@ def parse_date(string, option)
             say_error("Invalid date `#{string}` for option #{option} (expected format YYYY-MM-DD)")
             exit(1)
           end
+
+          def bundle_install(path, sha)
+            opts = {}
+            opts[:chdir] = path
+            out, status = Open3.capture2e("bundle install", opts)
+            unless status.success?
+              say_error("Can't run `bundle install` for commit #{sha}")
```

Comment:
I think we can give more info here to say something like 

```
"Can't run `bundle install` for commit #{sha}. Coverage snapshot is skipped"
```

Reply:
Done.